src/System.Net.HttpListener/tests/AuthenticationTests.cs

```diff
@@ -146,7 +154,8 @@ public static IEnumerable<object[]> BasicAuthenticationHeader_TestData()
             await ValidateNullUser();
         }
 
-        [ConditionalFact(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementationAndNotUap))] // [PlatformSpecific(TestPlatforms.Windows, "Managed impl doesn't support NTLM")]
```

Comment:
Could you put back the comment about why it's only for the Windows implementation?